tv/pages/rooms/index.tsx

```diff
@@ -0,0 +1,26 @@
+import { getRooms } from '../../api/queries/getRooms';
+import { Room } from '../../interfaces';
+import RoomList from '../../components/RoomList';
+
+type RoomProps = {
```

Comment:
Should it be plural (`RoomsProps`)?

Reply:
Updated in speaker details branch

Reply:
It does not seem to have been updated there.